src/lib/display-mobile.ts

```diff
@@ -1,6 +1,8 @@
-export function isMobile() {
-  const displayMobile =
-    window.innerWidth < 450 ||
-    /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent)
-  return displayMobile
+import { detect } from 'detect-browser'
```

Comment:
@bludnic @PaulDremanovich 
Let's choose a single function to check `isMobile`.
See https://github.com/Adamant-im/adamant-im/pull/601